examples/js/loaders/GLTFLoader.js

```diff
@@ -1613,6 +1613,11 @@ THREE.GLTFLoader = ( function () {
 
 				pending.push( parser.assignTexture( materialParams, 'normalMap', material.normalTexture.index ) );
 
+				if ( material.normalTexture.scale ) {
```

Comment:
Technically it could, sure. Let's do `material.normalTexture.scale !== undefined` then?